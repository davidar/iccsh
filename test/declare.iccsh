:declare fn void (*fn)(void)
:info fn

:declare coords
struct {
    int x, y;
} coords = { 4, 2 }

:info coords
:printf %d,%d coords.x, coords.y

const int x = 42
:info x
:printf %d x
