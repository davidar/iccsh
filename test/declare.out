void (*fn)(void)
struct {
    int x, y;
} coords
4,2
const int x
42
